web/tests/drag-select.spec.ts

```diff
@@ -4,26 +4,26 @@ import { Workspace } from './lynxkite';
 
 let workspace: Workspace;
 test.beforeAll(async ({ browser }) => {
-    workspace = await Workspace.empty(browser);
-    await workspace.addBox({ id: 'ex0', name: 'Create example graph', x: 100, y: 100 });
-    await workspace.addBox({
-      id: 'pr1',
-      name: 'Compute PageRank',
-      x: 100,
-      y: 200,
-      after: 'ex0',
-    });
-    await workspace.addBox({
-      id: 'pr2',
-      name: 'Compute PageRank',
-      x: 100,
-      y: 300,
-      after: 'pr1',
-    });
+  workspace = await Workspace.empty(await browser.newPage());
+  await workspace.addBox({ id: 'ex0', name: 'Create example graph', x: 100, y: 100 });
```

Comment:
Sorry about the whitespace change! We're using 2 spaces for tabs. If this needs to be set in some Lint config, please make the change.